fakeredis/_server.py

```diff
@@ -2129,6 +2129,20 @@ def zincrby(self, key, increment, member):
     def zlexcount(self, key, min, max):
         return key.value.zlexcount(min.value, min.exclusive, max.value, max.exclusive)
 
+    @command((Key(ZSet),), (Int,))
+    def zpopmin(self, key, count=None):
```

Comment:
Unfortunately the `@command` decorator doesn't support optional (0 or 1 instance) arguments yet, only repeating arguments (0 or more instances), so you need to make the argument a `*args` and then check in the function if there are too many arguments (see `_zrangebylex` for an example).

This will also need a unit test, which can be done with `r.execute_command` to directly pass parameters.